srcpkgs/ardour/template

```diff
@@ -1,22 +1,47 @@
 # Template file for 'ardour'
 pkgname=ardour
-version=6.0
+version=6.3
 revision=1
+wrksrc="Ardour-${version}.0"
```

Comment:
Why not add zero to version?

Reply:
I followed there tag schema, will add 0 into it.